File: Data Structures & Algorithms/is-anagram/submission-0.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        s1 = []
        t1 = []
        for char in s:
            s1.append(char)
        for char in t:
            t1.append(char)
        s1.sort()
        t1.sort()
        if(s1 == t1):
            return True
        else: 
            return False

File: Data Structures & Algorithms/subsets-ii/submission-0.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {number[][]}
     */
    subsetsWithDup(nums) {
         let res = [[]]
        for(let num of nums){
            let length = res.length
            for(let i = 0; i<length; i++){
                let cur = [...res[i], num].sort().toString()
                let isValid = true
                for(let arr of res){
                    if (cur == arr.sort().toString()){
                        isValid=false
                    }
                }
                if(isValid){
                    res.push([...res[i], num])
                }
            }
        }
        return res
    }
}
